lms/templates/widgets/segment-io-footer.html

```diff
@@ -0,0 +1,14 @@
+% if settings.LMS_SEGMENT_KEY:
+    <!-- begin segment footer -->
```

Comment:
I think @andy-armstrong originally only looked at base.html and main.html. Good catch about footer.html and footer-edx-v3.html, I will check those out.